src/validatePrTitle.js

```diff
@@ -34,7 +34,7 @@ module.exports = async function validatePrTitle(
 
   if (!result.type) {
     throw new Error(
-      `No release type found in pull request title "${prTitle}". Add a prefix to indicate what kind of release this pull request corresponds to (see https://www.conventionalcommits.org/).\n\n${printAvailableTypes()}`
+      `No release type found in pull request title "${prTitle}". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/.\n\n${printAvailableTypes()}`
```

Comment:
It seems like the dot is also part of the anchor:

![Screenshot 2021-07-26 at 09 43 12](https://user-images.githubusercontent.com/4038316/126951706-0e02b888-bdd4-4f05-aefb-38e985788fbc.png)

The navigation works nonetheless, but maybe to be on the safe side we should remove it.

```suggestion
      `No release type found in pull request title "${prTitle}". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/\n\n${printAvailableTypes()}`
```